.github/CODEOWNERS

```diff
@@ -308,6 +308,8 @@
 /core/templates/dev/head/components/button-directives/create-activity-button.directive.html @nithusha21
 /core/templates/dev/head/components/common-layout-directives/navigation-bars/ @nithusha21
 /core/templates/dev/head/services/navigation.service*.ts @nithusha21
+/core/templates/dev/head/services/navigation-bars-backend-api.service.spec.ts @nithusha21 @cdrebin
```

Comment:
Make this `/core/templates/dev/head/services/navigation-bars-backend-api.service*.ts @nithusha21` and remove the next line.